com.obsproject.Studio.Plugin.waveform.yml

```diff
@@ -0,0 +1,39 @@
+id: com.obsproject.Studio.Plugin.waveform
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//5.15-21.08
+# Build fails with GCC, we have to use Clang
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm13
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/waveform
+  append-path: /usr/lib/sdk/llvm13/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm13/lib
+
+modules:
+  - name: waveform
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_C_COMPILER=clang
+      - -DCMAKE_CXX_COMPILER=clang++
+      - -DPACKAGED_INSTALL=ON
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo pkg/${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
+      - type: git
+        url: https://github.com/phandasm/waveform.git
+        tag: v1.3.0-flatpak
+        commit: d5eb5359c672a8b44b47a8feb28cfe0a3babd992
+        # Re-enable FEDC after the next release
```

Comment:
Why not now? :slightly_smiling_face:

Reply:
Because `v1.3.0` was released before `v1.3.0-flatpak` (which includes changes to allow building it without patches), and as is FEDC will try to update to the unbuildable `v1.3.0`. I'll uncomment when there's a `v1.3.1` or `v1.4.0` upstream to not have a useless PR sitting on the repo :)

Reply:
Alright! Thanks for clarifying that. I have no other comments, this looks ready to me.

Reply:
Fedc never opens the same PR twice, so you can let it open the downgrade, close it, and it'll keep quiet till the next release

Reply:
> Fedc never opens the same PR twice, so you can let it open the downgrade, close it, and it'll keep quiet till the next release

oh lol. I can point many times where it opened the PR multiples times.

Reply:
Please do! https://github.com/flathub/flatpak-external-data-checker/blob/71db1f193d946180d7354babac5168f3460bf156/src/main.py#L162-L167 and https://github.com/flathub/flatpak-external-data-checker/blob/71db1f193d946180d7354babac5168f3460bf156/src/main.py#L275-L284 are meant to ensure that this doesn't happen. The branch name is based on the tree hash, so identical tree contents should not be proposed twice.